cmd/cluster/cluster_create.go

```diff
@@ -96,8 +96,17 @@ func (o *CreateOptions) Run(cmd *cobra.Command) error {
 	o.InitializeProgressBar(cmd.OutOrStdout())
 	o.saveData.setAgentIdentifier(createSandboxRequest.AgentIdentifier)
 	o.saveData.setCreateSandboxResponse(*sandboxResponse)
+	var configErr *configuration.ConfigError
+	attempts := 0
 	for {
 		cluster, err := o.ArmoryClient.Sandbox().Get(ctx, o.saveData.getClusterId())
+		if errors.As(err, &configErr) && attempts < 3 {
```

Comment:
I thought we where using: https://github.com/eapache/go-resiliency or https://github.com/hashicorp/go-retryablehttp in our code bases.

Reply:
https://github.com/armory-io/armory-cli/blob/main/pkg/armoryCloud/client.go#L39

:point_up: could possibly be an instance of :point_down: 

https://github.com/armory-io/go-commons/blob/c916a06372fc1f8b357871e6159ed0a084c1ea75/wormhole/proxy/wormhole.go#L51-L58

Reply:
Yes, yes. You are right. I was being dense. Thank you for the helpful links!